components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/db/PinnedSiteDao.kt

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.top.sites.db
+
+import androidx.annotation.WorkerThread
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+
+/**
+ * Internal DAO for accessing [PinnedSiteEntity] instances.
+ */
+@Dao
+internal interface PinnedSiteDao {
```

Comment:
Note for future: have some top sites from an old build. Install a new build with this PR and ensure the database doesn't need to have it's schema migrated if we change the dao interface naming.